Library/ENV/4.3/cc

```diff
@@ -208,7 +207,7 @@ class Cmd
     return keep_orig?(path) unless ENV["HOMEBREW_EXPERIMENTAL_FILTER_FLAGS_ON_DEPS"]
 
     # Allow references to self
-    if keg_path && path.start_with?(keg_path)
+    if formula_prefix && path.start_with?(formula_prefix)
```

Comment:
True. May as well do that, since the "`1.2.3` vs `1.2.3_1`" edge case is really close to the edge this PR is fixing in the first place. Amended.